lib/test.js

```diff
@@ -165,35 +172,49 @@ Test.prototype._end = function (err) {
 Test.prototype.exit = function () {
 	var self = this;
 
-	// calculate total time spent in test
-	this.duration = globals.now() - this._timeStart;
+	function checkAssertLength() {
```

Comment:
I would make it a private instance method (prefixed with `_`). The function checks if assertion count matches plan count. I think the name could be clearer.